src/types/responses/ForeignAssets.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import { AssetMetadata } from '@polkadot/types/interfaces';
+import { PalletAssetsAssetDetails } from '@polkadot/types/lookup';
+import { AnyJson } from 'src/types/polkadot-js';
+
+import { IAt } from '.';
+
+export interface IForeignAssetInfo {
+	foreignAssetInfo: PalletAssetsAssetDetails | AnyJson;
```

Comment:
These types should have `AnyJson` removed from them

Reply:
I replaced with `T`/`Codec` like I had it in the beginning. If I remove completely and only leave types `PalletAssetsAssetDetails` and `AssetMetadata` it does not work.